src/graphics/gfx_message.cpp

```diff
@@ -34,19 +39,36 @@ void DrawMessage(const std::string& SuperText)
     int lineStart = 0; // start of current line
     int lastWord = 0; // planned start of next line
     int numLines = 0; // n lines
-    int maxChars = (TextBoxW-24)/charWidth+1; // 27 by default
+    const int maxChars = ((TextBoxW - 24) / charWidth) + 1; // 27 by default
+
+    // Build map
+    std::vector<const char*> SuperTextMap;
```

Comment:
Better to do this when entering the message pause state I think instead of allocing and freeing every frame

Reply:
I optimised this just now, please review.